### ISSUE TITLE
HYPERVISOR_ERROR after Windows update

### ISSUE THREAD

**Issue by ad-on-is** [microsoft/WSL#5288]
I don't know if this is realted to WSL or not, but since a view Windows versions back, I am getting a green screen when updating Windows. The error message is HYPERVISOR_ERROR.

Althouh, after I force-restart my PC the update finishes successfully and everything works just fine, I'm still worried that something more serious and unexpected might happen in the future.

**Comment by onomatopellan** [comment]
What's the CPU?

**Comment by ad-on-is** [comment]
> What's the CPU?
i7-8750H

**Comment by onomatopellan** [comment]
And you see the error only during the build install and then never again until next build install? I guess disabling Virtualization on BIOS would prevent the GSOD but you will need to enable it again for WSL2.

**Comment by ad-on-is** [comment]
@onomatopellan Exaclty, the last reboot of the build install causes the GSOD. Toggeling virtualization in BIOS is not the solution, since I never had this issue before. It started 3-4 builds ago.

**Comment by onomatopellan** [comment]
If it's on last reboot then it could be a driver that crashes with virtualization enabled. You could look at Event viewer if there was some error by that time.

**Comment by therealkenc** [comment]
> The error message is HYPERVISOR_ERROR.

I've seen one of these second hand. You can try sending a minidump per the template if have one. Usually (historically) GSODs do sort themselves in forthcoming insider builds eventually (often pretty quick).

> Please direct all Windows crashes and security issues to secure@microsoft.com.  Ideally, please [configure your machine to capture minidumps](https://support.microsoft.com/en-us/help/315263/how-to-read-the-small-memory-dump-file-that-is-created-by-windows-if-a), repro the issue, and send the minidump from "C:\Windows\minidump\".\\

**Comment by ad-on-is** [comment]
@therealkenc `Automatic memory dump` was already enabled in the settings. The `MEMORY.DMP` is 2GB. I assume, I should change it to `Small memory dump`, wait until the next GOSOD happens and then provide that 256kb file to Microsoft, right?

BTW: Yesterday I installed 2 builds, the first one resulted in a GSOD, the second one did not.

**Comment by therealkenc** [comment]
>  and then provide that 256kb file to Microsoft, right?

Yes, if you'd like.

**Comment by onomatopellan** [comment]
I think this is new. Helps to diagnostic faulty installs.
https://docs.microsoft.com/en-us/windows/deployment/upgrade/setupdiag

`SetupDiag.exe /Output:C:\Results.log`

**Comment by keesvanbochove** [comment]
I have a HYPERVISOR_ERROR GSoD occassionally on build 19041, this happens since I've enabled Windows Insider Program and downloaded this build. I have enabled WSL2 and installed Docker, but I even get the error when Docker is not running. Other people seem to have similar issues (https://www.reddit.com/r/windowsinsiders/comments/gkph4y/gsod_with_hypervisor_error_but_only_for_the_first/). My processor is an AMD Ryzen 9 3950X.

**Comment by msarson** [comment]
I am having the same issue as well for the last few fast ring flight builds.  Intel Core i7-9700K, docker running inside WSL2

As reported, the first try at installing a new insider gives the green screen, but you get a successful update on the second attempt.

**Comment by nithinmohantk** [comment]
I am having same issue with Build 19635, it is not updating further as every windows update crashes with BSOD - "HYPERVISOR_ERROR" , I too have docker for Windows with WSL enabled.

**Comment by ivanmonteiro** [comment]
I'm insiders fast ring build 19640, wsl 2 enabled and docker desktop.
I did not have this error while I was on slow ring build 19041.
I'm having a green screen of death with HYPERVISOR ERROR. It hapens on first boot, right after login screen appears or in the first minutes of usage. After a reboot, the computer works flawlessly without any green screen of death.
So basically it happens on 90% of first boot of a day, and it almost certainly does not hapen on the next couple of boots at the same day.
My laptop is a Dell g3 i7 8750H.

**Comment by patricknazar** [comment]
yes I also have the same issue. First boot of the day, even if i've been up overnight and restart the computer that morning. Windows insiders fast ring 2004 build 19640.1

One way with dealing with this problem is when I see that docker isn't starting, or when it pops up giving me a seemingly different but similar error (can't connect, no response, etc) I just preemptively restart the computer and docker will start upon boot and I won't get the sudden devastation of having my work taken away from me with a GSOD

**Comment by ad-on-is** [comment]
> I'm having a green screen of death with HYPERVISOR ERROR. It hapens on first boot, right after login screen appears or in the first minutes of usage. After a reboot, the computer works flawlessly without any green screen of death.
> So basically it happens on 90% of first boot of a day, and it almost certainly does not hapen on the next couple of boots at the same day. On the next day, however, there is a 90% chance of getting a GSOD on the first boot.

The only GSOD I see is as I already described, only while updating. After that, I never experience GSODs again, no matter how often I restart the PC.

**Comment by patricknazar** [comment]
I got one during an update to 19645.1 last night, same error. Luckily the pc wasn't "bricked" and i just had to restart the update upon restart for it to work

**Comment by ivanmonteiro** [comment]
> > What's the CPU?
> 
> i7-8750H

@ad-on-is  I have the same CPU on my notebook and I'm also having green screen of death with hypervisor error usually on first boot.

**Comment by rafaels-dev** [comment]
I also have same issue. My processor is i5-4440. The error started after I updated WSL Kernel for use Docker Desktop with WSL2 backend.
I uninstalled docker, but my WSL2 (Ubuntu) become unstable. Terminal freeze on some commands and every time I try to restart WSL using `wsl --shutdown` command, the green screen appears with HYPERVISOR_ERROR.
![image](https://user-images.githubusercontent.com/5066597/84595994-689d6f00-ae31-11ea-861e-410fcb02418d.png)

**Comment by rafaels-dev** [comment]
> I also have same issue. My processor is i5-4440. The error started after I updated WSL Kernel for use Docker Desktop with WSL2 backend.
> I uninstalled docker, but my WSL2 (Ubuntu) become unstable. Terminal freeze on some commands and every time I try to restart WSL using `wsl --shutdown` command, the green screen appears with HYPERVISOR_ERROR.
> ![image](https://user-images.githubusercontent.com/5066597/84595994-689d6f00-ae31-11ea-861e-410fcb02418d.png)

I did "reset" on my Ubuntu following the procedure described in this article and ubuntu terminal become stable again: https://petri.com/how-to-uninstall-and-reset-windows-subsystem-for-linux-distributions

**Comment by ivanmonteiro** [comment]
I just downloaded updated drivers and reinstalled all drivers, with no success. Also got the latest version of wsl kernel. Same hypervisor error happens on first boot of the day.
Now I uninstalled docker desktop, all wsl distros installed by the windows store, disabled the windows features of WSL and virtualization plataform. 
And I'm testing now re-enabling wsl 2 and installed docker desktop again. I will report back in a couple of days if it solved the Green Screen Of Death.

**Comment by dparker2** [comment]
I am getting this gsod error as well on build 20161. I do not use docker for windows since I have a home edition. However, I did recently upgrade the WSL 2 kernel, so it may have to do with that.

**Comment by 0nOff** [comment]
Disabling the hypervisor helped me (win 10 pro 2004 20161.1000)

`Disable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V-Hypervisor`

**Comment by therealkenc** [comment]
There was some traffic in today's `20175` [announce](https://blogs.windows.com/windowsexperience/2020/07/22/announcing-windows-10-insider-preview-build-20175/):

![image](https://user-images.githubusercontent.com/7049700/88209750-068e1180-cc08-11ea-8b68-ab8b069c3534.png)

That might be us. If folks still experience the `HYPERVISOR ERROR` GSOD in `20175` or better (but not before) please add a comment to help track whether we can call this nominally addressed. [Noting it is tough to prove a negative.]

**Comment by ad-on-is** [comment]
@therealkenc I just upgraded to 20175 and got the HYPERVISOR_ERROR again. Is this supposed to 
happen with this update, while all future updates should be fixed? Or shouldn't it even happen on this update?

**Comment by therealkenc** [comment]
Hmm... I _suppose_ one could be an optimist and _plausibly_ imagine one last GSOD hurrah before the fix was actually applied by the update. But I wouldn't deign to give odds on that until the next roll. Thanks for the feedback: very good to know either way.

**Comment by ivanmonteiro** [comment]
> @therealkenc I just upgraded to 20175 and got the HYPERVISOR_ERROR again. Is this supposed to
> happen with this update, while all future updates should be fixed? Or shouldn't it even happen on this update?

If hypervisor error persists, 
try uninstalling any distros (and docker desktop if you use it) and disabling virtualization platform and wsl on Windows features.
Then restart and install wsl again.
This worked for me.

**Comment by therealkenc** [comment]
> This worked for me.

Noting it is tough to prove a negative (an _absence_ of BSOD).

> try uninstalling any distros

There is no particular reason why uninstalling _especially_ a distro would have _any_ effect whatso on a kernel bugcheck (it wouldn't). Likewise while you've got nothing to lose (in the hydroxychloroquine sense) by doing a disable/enable Virtual Machine Platform rain dance, there is no particular reasoning to support doing that would actually help (versus whatever the 20175 Windows Update does by itself).

**Comment by ivanmonteiro** [comment]
> > This worked for me.
> 
> Noting it is tough to prove a negative (an _absence_ of GSOD).
> 
> > try uninstalling any distros
> 
> There is no particular reason why uninstalling _especially_ a distro would have _any_ effect whatso on a kernel bugcheck (it wouldn't). Likewise while you've got nothing to lose (in the hydroxychloroquine sense) by doing a disable/enable Virtual Machine Platform rain dance, there is no particular reasoning to support doing that would actually help (versus whatever the 20175 Windows Update does by itself).

You have a point about that. 
I'm on older windows version, and what can I say, It worked for me. 
I was having daily GSOD with hypervisor_error code and removing distros, disabling WSL on windows features and re-installing it stopped those daily GSODs I was having. This was the only change I made to my system that actually stopped those GSODs.

Remember it will take a while until everybody gets the new build 20175.

**Comment by patricknazar** [comment]
While it would seem I am no longer getting this (I'll update if I do), wsl2 still doesn't work "first boot of the day" and I have to restart the computer. Previously I could tell that the GSOD was coming as nothing wsl2 would start, I'd restart and all was well (for the most part). Now, I still have to restart the computer to get wsl2 working. Docker-desktop gives me `stdout: The operation timed out because a response was not received from the virtual machine or container.` and Ubuntu just sits in black window i.e. won't start. There is something else going on and it's related to this, somehow.

**Comment by rlperez** [comment]
Presumably this will be fixed soon? This has gotten worse after I moved to an insider build hoping for a fix as it's been an ongoing issue already but it's been exponentially getting worse. My computer barely runs for 10 minutes before blowing up. I'm going to have to clean install to another operating system since I keep crashing all over the place including in the middle of meetings with this specific problem. First boot of the day is the least of the problems if it crashes by time I get all my software in order. The only time the system is remotely stable (and it's still not on this build perfect as I expected) is if hypervisor is off at a hardware level. Disabling the additional options in the software made no impact. I'm using WSL2. Had to reboot 3 times in the time it took me to type this. 

Last boot crashed before I could even finish typing the build number. 

Build 20241
AMD Ryzen 5 3600

Woops. Happened again before I could hit send.

**Comment by alexbond73** [comment]
I have Dev channel preview Windows 10 Pro build 21354.1 installed, to be able to run Hyper-V with  AMD Ryzen 5 series CPU.
The VM was created 'version 10', from PowerShell (Admin) as:
```
New-Vm -VMName "Win10 10" -Version 10.0 -Generation 2 -MemoryStartupBytes 10220MB
Set-VMProcessor "Win10 10" -Count 6 
Add-VMHardDiskDrive -VMName "Win10 10" -Path "Path\VMdisk.vhdx"
Set-VMProcessor "Win10 10" -ExposeVirtualizationExtensions $true
```
I needed the _nested virtualization_ on, for the sake of running WSL2 linux. 
Unfortunately I was getting HYPERVISOR_ERROR and a complete halt of the host machine, with the Green Screen of Death, and 0% progress collecting debug data.
The VM became stable only after switching off the _nested virtualization_ feature.
```
Set-VMProcessor "Win10 10" -ExposeVirtualizationExtensions $false
```

**Comment by cicorias** [comment]
hmm, this is happening on a brand new Surface Laptop 4, i7/32gb/1TB 15"... 

Be great to get an update. I'm NOT on Insiders 

![image](https://user-images.githubusercontent.com/1080406/116800222-173fb200-aacd-11eb-9a07-db6d330190d3.png)

**Comment by SimonHofmann1** [comment]
@cicorias  I got exactly the same error with my brand new Surface Laptop 4. With or without Hyper-V enabled. All Surface Checks are Fine with the diagnostigs Tool and with the Memtest. I got this BSOD 3 Times dayli .  
I tried with the Cloud Reset  and Local Wipe&Load with 20H1 and 20H2. Even with windows 11 I got the same error every time. 
Did you find a solution? Is there a Case opened by Microsoft? 
![image](https://user-images.githubusercontent.com/45968420/126172485-00c5aed8-b8d3-44d6-b762-75c2b3d718cb.png)

**Comment by cicorias** [comment]
> @cicorias I got exactly the same error with my brand new Surface Laptop 4. With or without Hyper-V enabled. All Surface Checks are Fine with the diagnostigs Tool and with the Memtest. I got this BSOD 3 Times dayli .
> I tried with the Cloud Reset and Local Wipe&Load with 20H1 and 20H2. Even with windows 11 I got the same error every time.
> Did you find a solution? Is there a Case opened by Microsoft?
> ![image](https://user-images.githubusercontent.com/45968420/126172485-00c5aed8-b8d3-44d6-b762-75c2b3d718cb.png)

I have a problem report open; I have not seen an update nor was there a fix.

I'll follow up and see if they have an update. Give me some time.

**Comment by totalgaara** [comment]
Same issues on Windows 11 with my AMD FX 8320.  Already made an issue, craigloewen was kind to try helping me, my bsod have been sent to Hypervisor Teams, they said that my CPU has an internal issue.. but's that not the case, WSL / Hyper -V in general works fine on W10 on my side, so i don't see any issues with my processor...

**Comment by SimonHofmann1** [comment]
[MiniDumps.zip](https://github.com/microsoft/WSL/files/6907758/MiniDumps.zip)

Hypervisior_Error don't come alone. It's like a BSOD Bingo with the 4 following outputs:

HYPERVISOR_ERROR (20001)
INTERRUPT_EXCEPTION_NOT_HANDLED (3d)
DRIVER_IRQL_NOT_LESS_OR_EQUAL (d1)
MEMORY_MANAGEMENT (1a)
    # Any other values for parameter 1 must be individually examined.
    
In the ZIP File are some Minidumps from the last few days.. happens up to 5 Times dayli.
Very frustrating to work with this Surface Laptop 4. It happens on AC/DC, with External LG USB-C Monitor and without External LG Screen. Really a BSOD Bingo, can happens with any circumstances.

**Comment by Larisho** [comment]
@SimonHofmann1 I'm getting the exact same thing except that the screen hangs at 100% and never writes a dump file.

**Comment by JonathanVQP** [comment]
I am running Windows 11 Build 22000.184 for my amd processor and this issue is still not resolved. If you don't enable WSL2 features and hypervisor, this build runs great without it. However, if you do install WSL2 and hypervisor issues, you get the GSOD with the hypervisor error!  Even after the error shows up and you reboot, the Windows 11 runs very slow and will crash intermittently. I don't know if this is due to AMD processors specifically or what....!

**Comment by dhanar10** [comment]
> @SimonHofmann1 I'm getting the exact same thing except that the screen hangs at 100% and never writes a dump file.

I believe I experienced the same issue. Windows 10 fully updated as of today on Lenovo laptop with Ryzen CPU.

**Comment by zx2c4** [comment]
```
6: kd> !analyze -v
*******************************************************************************
*                                                                             *
*                        Bugcheck Analysis                                    *
*                                                                             *
*******************************************************************************

HYPERVISOR_ERROR (20001)
The hypervisor has encountered a fatal error.
Arguments:
Arg1: 0000000000000011
Arg2: 0000000000307cf9
Arg3: 0000000000001004
Arg4: 0000010002005800

Debugging Details:
------------------


KEY_VALUES_STRING: 1

    Key  : Analysis.CPU.mSec
    Value: 1952

    Key  : Analysis.DebugAnalysisManager
    Value: Create

    Key  : Analysis.Elapsed.mSec
    Value: 1921

    Key  : Analysis.Init.CPU.mSec
    Value: 1374

    Key  : Analysis.Init.Elapsed.mSec
    Value: 8427

    Key  : Analysis.Memory.CommitPeak.Mb
    Value: 86

    Key  : WER.OS.Branch
    Value: co_release

    Key  : WER.OS.Timestamp
    Value: 2021-06-04T16:28:00Z

    Key  : WER.OS.Version
    Value: 10.0.22000.1


DUMP_FILE_ATTRIBUTES: 0x800

BUGCHECK_CODE:  20001

BUGCHECK_P1: 11

BUGCHECK_P2: 307cf9

BUGCHECK_P3: 1004

BUGCHECK_P4: 10002005800

BLACKBOXBSD: 1 (!blackboxbsd)


BLACKBOXNTFS: 1 (!blackboxntfs)


BLACKBOXPNP: 1 (!blackboxpnp)


BLACKBOXWINLOGON: 1

PROCESS_NAME:  System

BAD_STACK_POINTER:  ffff930036206ca8

STACK_TEXT:  
ffff9300`36206ca8 fffff806`5b3474db     : 00000000`00020001 00000000`00000011 00000000`00307cf9 00000000`00001004 : nt!KeBugCheckEx
ffff9300`36206cb0 fffff806`5b257c36     : 00000000`00000000 fffff806`5b00b05c ffff9300`360fc3c0 ffff9300`36206d30 : nt!HvlSkCrashdumpCallbackRoutine+0x6b
ffff9300`36206cf0 fffff806`5b21eb82     : 00000000`00000000 ffff9300`36206ef0 00000000`00000000 00000000`00000000 : nt!KiProcessNMI+0x24d666
ffff9300`36206d30 fffff806`5b21e903     : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : nt!KxNmiInterrupt+0x82
ffff9300`36206e70 fffff806`5b1964cd     : 00000000`00000000 00000000`0ad7d9f2 00000000`00000000 0000006c`fe775d5e : nt!KiNmiInterrupt+0x243
fffff883`5547f510 fffff806`5b0eaa41     : 00000000`0000029c ffff9300`36180100 00000000`00000000 00000000`0000029c : nt!PpmIdleGuestExecute+0x1d
fffff883`5547f550 fffff806`5b0ea256     : 00000000`00000000 fffff806`5b12b07a ffffffff`ffffffb1 fffff806`5b219ced : nt!PpmIdleExecuteTransition+0x661
fffff883`5547f9f0 fffff806`5b216a74     : 00000000`00000000 ffff9300`3618c440 ffffd286`9caad040 00000000`000005f4 : nt!PoIdle+0x3a6
fffff883`5547fbc0 00000000`00000000     : fffff883`55480000 fffff883`55479000 00000000`00000000 00000000`00000000 : nt!KiIdleLoop+0x54


SYMBOL_NAME:  nt!HvlSkCrashdumpCallbackRoutine+6b

MODULE_NAME: nt

IMAGE_NAME:  ntkrnlmp.exe

STACK_COMMAND:  .thread ; .cxr ; kb

BUCKET_ID_FUNC_OFFSET:  6b

FAILURE_BUCKET_ID:  0x20001_11_307cf9_STACKPTR_ERROR_nt!HvlSkCrashdumpCallbackRoutine

OS_VERSION:  10.0.22000.1

BUILDLAB_STR:  co_release

OSPLATFORM_TYPE:  x64

OSNAME:  Windows 10

FAILURE_ID_HASH:  {b3065167-1c4a-54a5-1ca5-d1a56b1755a3}

Followup:     MachineOwner
---------

```

**Comment by dhanar10** [comment]
By the way, if you are using Lenovo laptop with AMD Ryzen CPU, you might get hit by this issue instead:

https://answers.microsoft.com/en-us/windows/forum/all/legion-5-15arh05h-multiple-bsods-per-day/9b6bb4a9-2329-4907-af42-774e8deeb32f

https://forums.lenovo.com/t5/Gaming-Laptops/Legion-5-15ARH05-constant-abrupt-restarts/m-p/5047613?page=2#5325241

The BSOD messages is matching this issue.

TLDR The workaround is https://www.reddit.com/r/ZephyrusG14/comments/gho535/important_update_to_properly_disable_boosting/fqzjbvm?utm_medium=android_app&utm_source=share&context=3

After implementing the workaround, I can use WSL2 normally without any hypervisor error BSOD. It works at least on my Lenovo laptop with Ryzen CPU.

**Comment by leedonhyun** [comment]
![image](https://user-images.githubusercontent.com/8449892/146634076-ce6c7747-f156-4a93-bb01-76d7c858be6b.png)


HYPERVISOR_ERROR (20001)
The hypervisor has encountered a fatal error.
Arguments:
Arg1: 0000000000000011
Arg2: 000000000028f186
Arg3: 0000000000001005
Arg4: ffffe80001ad1a50

Debugging Details:
------------------


KEY_VALUES_STRING: 1

    Key  : Analysis.CPU.mSec
    Value: 4249

    Key  : Analysis.DebugAnalysisManager
    Value: Create

    Key  : Analysis.Elapsed.mSec
    Value: 15145

    Key  : Analysis.Init.CPU.mSec
    Value: 4921

    Key  : Analysis.Init.Elapsed.mSec
    Value: 34578

    Key  : Analysis.Memory.CommitPeak.Mb
    Value: 89

    Key  : WER.OS.Branch
    Value: vb_release

    Key  : WER.OS.Timestamp
    Value: 2019-12-06T14:06:00Z

    Key  : WER.OS.Version
    Value: 10.0.19041.1


BUGCHECK_CODE:  20001

BUGCHECK_P1: 11

BUGCHECK_P2: 28f186

BUGCHECK_P3: 1005

BUGCHECK_P4: ffffe80001ad1a50

PROCESS_NAME:  msedge.exe

BAD_STACK_POINTER:  ffffe200b4b3cc98

STACK_TEXT:  
ffffe200`b4b3cc98 fffff804`194f3c0b     : 00000000`00020001 00000000`00000011 00000000`0028f186 00000000`00001005 : nt!KeBugCheckEx
ffffe200`b4b3cca0 fffff804`1950d6e8     : 00000000`00000001 00000000`00000002 ffffe200`b4b00180 fffff804`1951ffe0 : nt!HvlSkCrashdumpCallbackRoutine+0x6b
ffffe200`b4b3cce0 fffff804`19402782     : 8a000004`4913396b ffffe200`b4b3cef0 00000000`00000000 00000000`00000000 : nt!KiProcessNMI+0xe8
ffffe200`b4b3cd30 fffff804`19402552     : 8a000004`4913396b 00000000`00000000 00000000`00000000 00000000`00000000 : nt!KxNmiInterrupt+0x82
ffffe200`b4b3ce70 fffff804`1923d8a3     : 00000000`00000000 ffff9f96`47a00000 00000000`00001000 ffff8780`000081c0 : nt!KiNmiInterrupt+0x212
ffff8287`664595e0 fffff804`192f753b     : ffff8480`0cdb3990 8a000004`4913396b 00000000`d0000002 ffffcc8e`67358000 : nt!MiGetUltraMapping+0x33
ffff8287`66459650 fffff804`192f73ad     : ffff8480`0cdb3990 00000000`00000001 00000000`d0000002 00000000`00000002 : nt!MiMapPageInHyperSpaceWorker+0xbb
ffff8287`66459690 fffff804`1935d2d9     : ffff8480`0cdb3990 00000000`000001ff 00000000`00000000 00000000`00000200 : nt!MiFlushCacheForAttributeChange+0x45
ffff8287`664596c0 fffff804`193f1ca4     : 00000000`00000002 00000000`00000000 ffff8480`0cdb0000 fffff804`00000000 : nt!MiChangePageAttributeContiguous+0x19d
ffff8287`66459730 fffff804`193f0cea     : fffff804`19c50b80 00000000`00000001 00000000`00000000 ffff8e00`e4f46000 : nt!MiUnlinkNodeLargePages+0x86c
ffff8287`66459860 fffff804`19319e22     : ffff8287`66459980 fffff804`192a4302 00000000`00000000 00000000`00000001 : nt!MiGetFreeZeroLargePages+0x86
ffff8287`664598f0 fffff804`192a4f0d     : 00000000`00000001 00000000`00000000 00000000`00000000 fffff804`19c4f180 : nt!MiGetLargePage+0xc2
ffff8287`66459a40 fffff804`192b7e6c     : 00000000`0000001c 000001c9`e8c00000 ffffa988`f305b000 00000000`00000000 : nt!MiGetPageChain+0x21d
ffff8287`66459c80 fffff804`192a1dad     : ffff8287`66459d70 ffff8287`6645a210 ffffa988`f305c000 00000000`1c9e8e00 : nt!MiCreateSharedZeroPages+0x8c
ffff8287`66459d10 fffff804`192a8ba8     : ffff8e00`0000001c 00000000`00000001 ffff8480`06b591d0 00000000`00000002 : nt!MiResolveDemandZeroFault+0x27d
ffff8287`66459e00 fffff804`192a12b5     : ffff8287`66459f60 00000000`00000002 ffff8287`66459f40 ffffcc8e`8bd23740 : nt!MiResolveProtoPteFault+0x1338
ffff8287`66459f00 fffff804`1929f1f9     : 00000000`00000111 ffff8287`0000001c 00000000`c0000016 ffffa988`f305b000 : nt!MiDispatchFault+0x3d5
ffff8287`6645a040 fffff804`19345bec     : ffffcc8e`66c02000 00000000`00000600 fffff804`19000000 00000000`6c646d4d : nt!MmAccessFault+0x189
ffff8287`6645a1e0 fffff804`1924e6ec     : 0a000007`c091d867 ffff8e00`e4f46000 ffff8287`6645a340 ffff8e47`2391c8e0 : nt!MiFaultInProbeAddress+0xbc
ffff8287`6645a290 fffff804`1924da33     : 0a000007`c091d867 00000000`00000000 ffff8287`6645a3b9 000001c9`e8c00000 : nt!MiLockPageLeafPageTable+0x28c
ffff8287`6645a300 fffff804`1924d8c9     : ffffa989`00000000 00000000`00000000 00000000`c0000001 c0000001`d1dff86f : nt!MiProbeAndLockPages+0x153
ffff8287`6645a420 fffff804`4bd7cfc8     : ffffa989`0189dd10 ffffa989`0189dd10 00000000`005d1000 fffff804`4bd7eb6b : nt!MmProbeAndLockPages+0x29
ffff8287`6645a450 fffff804`4bd7fcd5     : ffffa988`f4d80230 ffffa988`f4d80230 ffff8287`6645a5b0 00000000`00000002 : dxgmms2!VidMmiProbeAndLockAllocation+0x100
ffff8287`6645a4f0 fffff804`4bd7ff08     : ffffa988`f4d80200 ffffcc8e`81596260 00000000`00000000 00000000`c0000017 : dxgmms2!VIDMM_GLOBAL::ProbeAndLockAllocation+0x85
ffff8287`6645a540 fffff804`4bda619a     : ffffa988`f4d80230 ffffa988`f4d80230 ffffa988`f4d80230 ffffa988`f4d80230 : dxgmms2!VIDMM_SEGMENT::LockAllocationBackingStore+0xc4
ffff8287`6645a600 fffff804`4bd87ff0     : ffffa988`f4d80230 ffffa988`f4d80230 ffffa988`f4d80230 ffffa988`f4d80230 : dxgmms2!VIDMM_APERTURE_SEGMENT::CommitResource+0xda
ffff8287`6645a6c0 fffff804`4bd86425     : ffffa989`00000000 ffff8287`6645a8e0 00000000`00000000 ffffa988`f4d80230 : dxgmms2!VIDMM_GLOBAL::PageInOneAllocation+0x1d0
ffff8287`6645a7e0 fffff804`4bd9c65e     : 00000000`00000002 00000000`00989680 ffffa989`02e03a30 00000000`00000001 : dxgmms2!VIDMM_GLOBAL::ProcessDeferredCommand+0xe25
ffff8287`6645aa00 fffff804`4bda6809     : ffffa989`02e03960 ffffcc8e`8be50001 00000000`00000000 00000000`3c68ac00 : dxgmms2!VIDMM_WORKER_THREAD::Run+0xe5e
ffff8287`6645abe0 fffff804`19355935     : ffffcc8e`8be50040 fffff804`4bda6800 ffffa989`02e03960 000f8067`b8bbbdff : dxgmms2!VidMmWorkerThreadProc+0x9
ffff8287`6645ac10 fffff804`193fe728     : fffff804`16f28180 ffffcc8e`8be50040 fffff804`193558e0 00000000`00000246 : nt!PspSystemThreadStartup+0x55
ffff8287`6645ac60 00000000`00000000     : ffff8287`6645b000 ffff8287`66455000 00000000`00000000 00000000`00000000 : nt!KiStartSystemThread+0x28


SYMBOL_NAME:  nt!HvlSkCrashdumpCallbackRoutine+6b

MODULE_NAME: nt

IMAGE_NAME:  ntkrnlmp.exe

STACK_COMMAND:  .thread ; .cxr ; kb

BUCKET_ID_FUNC_OFFSET:  6b

FAILURE_BUCKET_ID:  0x20001_11_28f186_STACKPTR_ERROR_nt!HvlSkCrashdumpCallbackRoutine

OS_VERSION:  10.0.19041.1

BUILDLAB_STR:  vb_release

OSPLATFORM_TYPE:  x64

OSNAME:  Windows 10

FAILURE_ID_HASH:  {7f445c4d-092b-a4f0-fbce-60c9be3f4876}

Followup:     MachineOwner
---------

**Comment by asvishnyakov** [comment]
Does anyone had a chance to solve issue? Looks like issue related to AMD processors

**Comment by razorbackdb** [comment]
After going back to windows 10 it stopped for me, disabling cpu boost in windows 11 solved it too but that means losing a lot of performance for anything else.
